### PR TITLE
Deployment README

### PR DESCRIPTION
**Pull request by peterfelts**
# Description and Motivation 

Deployment README

# Change type 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: deployment/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Node.js version 12.18.4 or newer
+- Azure subscription
+- Windows 10
+
+# Deployment
+Deployment is performed by providing the necessary configurations and then by running a Node.js deployment script. In order to get started: 
+1. Navigate to the 'deployment' directory
+2. Modify armtemplate/parameters.json to add required configuration values. 
+3. Run the following command (which requires Node.js to be installed):
+```nmp run deploy```
```
Comment (troyhopwood) [inline deployment/README.md:15]: Typo should be NPM nor NMP

**Review by troyhopwood — CHANGES_REQUESTED** [review]
quick typo should be fixed before checkin.

Later, should add info about the permissions required to both the sub and AAD as this will certainly trip people up.

File: deployment/README.md (hunk 1 above)
Comment (peterfelts) [reply deployment/README.md]: Fixed

**Review by saixiaohui — APPROVED** [review]

**Review by troyhopwood — APPROVED** [review]
lgtm

**Pull request merged**